homeassistant/helpers/config_validation.py

```diff
@@ -824,6 +826,29 @@ def make_entity_service_schema(
     )
 
 
+def _platform_validator(config):  # type: ignore
+    """Validate it is a valid platform."""
+    try:
+        platform = importlib.import_module(
+            f".{config[CONF_PLATFORM]}", "homeassistant.components.automation"
```

Comment:
Maybe not for this PR, but we should consider to move the triggers to `helpers/trigger.py` since they are now shared between automation and scripts.

Reply:
Yep, I realized I was partially duplicating some of the code from `automations/__init__.py`. So, yeah, that could use some reorganization now. But, agreed, that can happen in a follow on.